testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -186,11 +186,14 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
 
     def "distribution size should not change too much"() {
         expect:
-        def actual = (int) Math.ceil((double) getZip().size() / 1024 / 1024)
-        def expected = getDistributionSizeMiB()
+        def actualKB = (int) Math.ceil((double) getZip().size() / 1024)
+        def expectedKB = getDistributionSizeMiB() * 1024
```

Comment:
❓ Assuming the distribution is very close to 80.5MiB, do I put 80 or 81 as the expected size? And how likely is it to be just slight out of range for what I can configure?

I think getDistributionSizeMiB should return float (or double) and engineers should be instructed to set it to the "exact" value when it needs to be adjusted. That way one gets ± 0.5MiB (or 1MiB) tolerance

Reply:
I don't think I want to go down that path. How many digits are needed then?

I would propose the following alternative:
* Keep the size as an `int`
* Use the max of 1% of expected size or 1MB as the default margin
* Use 5 times that margin for release builds

That last change is the key one for release stability while protecting us against something going really bad between the promotion job and the building of the distribution.